docs_nnx/guides/gemma.ipynb

```diff
@@ -82,13 +90,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If everything went well, you should see:\n",
-    "```\n",
-    "Kaggle credentials set.\n",
-    "Kaggle credentials successfully validated.\n",
+    "If everything went well, it should say `Kaggle credentials set. Kaggle credentials successfully validated.`.\n",
+    "\n",
+    "**Note:** In Google Colab, you can instead authenticate into Kaggle using the code below after following the optional step 3 from above.\n",
+    "\n",
     "```\n",
+    "import os\n",
+    "from google.colab import userdata # `userdata` is a Colab API.\n",
+    "\n",
+    "os.environ[\"KAGGLE_USERNAME\"] = userdata.get('KAGGLE_USERNAME')\n",
+    "os.environ[\"KAGGLE_KEY\"] = userdata.get('KAGGLE_KEY')\n",
+    "``` \n",
     "\n",
-    "Now select and download the checkpoint you want to try. Note that you will need an A100 runtime for the 7b models."
```

Comment:
@cgarciae Since there are checkpoints and tokenizer files, changed to "model" instead of "checkpoint".

Reply:
Checking if free TPU v2-8 is sufficient.